lib/shopify.mjs

```diff
@@ -84,17 +109,45 @@ export const createShopifyCustomer = async ({
   } catch (error) {
     logger.verbose('An error was thrown while trying to create a customer.');
 
-    // If the customer was already created in Shopify, we get an error back.
-    // However, we don’t want to fail for that case, so check for it here.
-    customer = verifyCustomerExists(error);
+    // TODO refactor this, because it got pretty messy in here.
+    let customerResponse;
+    try {
+      customerResponse = await getCustomerByEmail(payload.customer.email);
+      logger.verbose(JSON.stringify(customerResponse.data.customers));
+    } catch (e) {
+      throw new Error('Unable to retrieve customers.');
+    }
+
+    // If the response came back empty, something went wrong with Shopify; bail.
+    if (
```

Comment:
May as well just use `lodash.get` here, but again, for the refactor :)

Reply:
Probably. One of my hangups is always _always_ trying to avoid lodash. 😅 But I should probably get over myself.

Reply:
@jlengstorf ha, I have the same hangup. I usually just do

```bash
yarn add lodash.get
```

so I don't need to import the entirety of it. We _need_ this -> https://github.com/tc39/proposal-nullish-coalescing